dev_guide/create_from_url.adoc

```diff
@@ -0,0 +1,81 @@
+[[dev-guide-create-from-url]]
+= Create From URL
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+[[overview]]
+== Overview
+
+Create from URL allows the construction of a URL that drops the user into the end
```

Comment:
This needs to me more clear. Maybe something like:

Create From URL is a function that allows you to construct a URL from an image stream, image tag, or template. (Not sure if this is an accurate rewrite, but not sure what is meant by "drops the user into"